Resources/input/fsh/extensions/PrescriptionDeliveryTypeEX.fsh

```diff
@@ -0,0 +1,9 @@
+Extension: PrescriptionDeliveryTypeEX
+Id: PrescriptionDeliveryTypeEX
+Description: "Defines types of types describing how prescription will be delivered."
+* insert Meta
+* value[x] only Coding
+* value[x] from PrescriptionDeliveryTypeVS (required)
+* value[x] ^binding.description = "Zustelltyp"
```

Comment:
Zeile 3 ist Englisch Zeile 7 ist Deutsch. Beides in Eng?

Reply:
Ja, die Domainsprache der Benutzer ist deutsch. Die Domainsprache der fsh-Files/Entwickler ist englisch.
Wir trennen (gedanklich) den Inhalt vom Medium. Manchmal sieht es absurd aus, ist meiner Meinung nach so korrekt.